spec/features/user_search_spec.rb

```diff
@@ -7,9 +7,16 @@
 
   describe 'without filter' do
     let!(:users) { FactoryBot.create_list(:user, 3) }
+    let(:second_user) { FactoryBot.create :user }
 
     before { visit search_users_path }
 
+    it 'is possible to filter by username' do
```

Comment:
Why did you add this test? Exactly this behaviour is tested in the line 55 (before your changes) / 62 (after your changes).
If you want to check that the first name is visible, then I would add this expectation to all tests below

Reply:
I removed the test.